.craft.yml

```
@@ -0,0 +1,17 @@
+minVersion: 0.33.2
+changelogPolicy: auto
+targets:
+  - name: maven
+    gradleCliPath: ./gradlew
+    mavenCliPath: scripts/mvnw
+    mavenSettingsPath: scripts/settings.xml
+    mavenRepoId: ossrh
+    mavenRepoUrl: https://oss.sonatype.org/service/local/staging/deploy/maven2/
+    android:
+      distDirRegex: /^(sentry-android-|.*-android).*$/
+      fileReplaceeRegex: /\d\.\d\.\d(-\w+(\.\d)?)?(-SNAPSHOT)?/
+      fileReplacerStr: release.aar
+  - name: github
+  - name: registry
+    sdks:
+      maven:io.sentry.kotlin.multiplatform:sentry-kotlin-multiplatform:
```

```suggestion
      maven:io.sentry:sentry-kotlin-multiplatform:
```

the group should be the same as for the java/android artifacts

---

Also, you would need to create an entry for each target platform, because these will be separate artifacts on maven, like:
```
maven:io.sentry:sentry-kotlin-multiplatform-android:
maven:io.sentry:sentry-kotlin-multiplatform-iosarm32:
maven:io.sentry:sentry-kotlin-multiplatform-iosarm64:
maven:io.sentry:sentry-kotlin-multiplatform-jvm:
// ... and so on
```